examples/foreach/foreach.bal

```diff
@@ -3,31 +3,34 @@ import ballerina/io;
 public function main() {
     io:println("Iterating over a string array:-");
     string[] fruits = ["apple", "banana", "cherry"];
-    // To fetch the values defined in the array, use one variable.
-    // To fetch the index and value, use two variables as comma separated values. E.g., `foreach i, v in fruits`.
+
+    // Foreach statement can be used to iterate over an array. Each iteration will return an element in the array. Note
```

Comment:
```suggestion
    // Foreach statement can be used to iterate over an array. Each iteration returns an element in the array. Note
```

Let's avoid future tense as much as possible. Use simple present instead. Normally the word "will" is indicative of future tense.